lib/segment/tests/integration/byte_storage_hnsw_test.rs

```diff
@@ -233,8 +233,8 @@ fn test_byte_storage_hnsw(
     for i in 0..attempts {
         let query = random_query(&query_variant, &mut rnd, dim);
 
-        let range_size = 40;
-        let left_range = rnd.gen_range(0..400);
+        let range_size = 160;
```

Comment:
I believe you changed that cause otherwise the quality were too bad, rigth?

Reply:
It's a `u8`-vectors test. This PR affects the full scan threshold in HNSW for `u8` and `f16` vectors. I changed the filter here because the old range has a small cardinality in terms of affected full scan

Reply:
Reverted this change and set `full_scan_threshold = 0` instead